onmt/Model.lua

```diff
@@ -80,15 +80,17 @@ function Model:initParams()
   local numParams = 0
 
   for i, key in ipairs(orderedIndex) do
-    params[i]:uniform(-self.args.param_init, self.args.param_init)
+    if params[i]:dim() > 0 then
```

Comment:
When the `Bridge` layer has no parameters (the default). It goes through this function like the `Encoder` and `Decoder` do.